modules/claims_api/app/models/concerns/claims_api/file_data.rb

```diff
@@ -27,6 +28,8 @@ def description
       def set_file_data!(file_data, doc_type, description = nil)
         if Flipper.enabled? :claims_load_testing
           ClaimsApi::V2::MockAwsService.new.store(file_data)
+        elsif Settings.claims_api.s3.mock
+          ClaimsApi::V2::MockBDUploaderService.new.store(file_data)
```

Comment:
Not sure I understand this. The set_file_data! method is called from PDF generator and saved as part of the claim data when uploading to s3, but is not called from the BD uploader, which uses the file data to retrieve the pdf and upload to BD. The intent of this PR is to mock AWS calls but not BD calls, correct?

Reply:
Looks like you reworked that with latest push...nevermind.